soccer/src/soccer/planning/planner_node.hpp

```diff
@@ -139,13 +143,45 @@ class PlannerForRobot {
     ~PlannerForRobot() = default;
 
 private:
+    /**
+     * @brief Create a PlanRequest based on the given RobotIntent.
+     *
+     * @details This is how gameplay's RobotIntents end up in the right planner (e.g.
+     * how a Pivot skill goes to PivotPath planner).
+     *
+     * @param intent RobotIntent msg
+     *
+     * @return PlanRequest based on input RobotIntent
+     */
     PlanRequest make_request(const RobotIntent& intent);
+
+    /*
+     * @brief Get a Trajectory based on the PlanRequest by ticking through all
+     * available planners.
+     *
+     * @details (Each planner implements an "is_applicable(PlanRequest.motion_command)").
+     *
+     * @param request PlanRequest that needs a motion plan made
+     *
+     * @return Trajectory (timestamped series of poses & twists) that satisfies
+     * the PlanRequest as well as possible
+     */
     Trajectory plan_for_robot(const planning::PlanRequest& request);
 
+    /*
+     * @brief Check that robot is visible in world_state and that world_state has been
+     * updated recently.
+     */
     [[nodiscard]] bool robot_alive() const;
 
+    /*
+     * @return true if current planner is done, false otherwise.
+     */
+    bool is_done() const;
```

Comment:
this should also be marked?
```suggestion
    [[nodiscard]] bool is_done() const;
```